string class_name

---

bool success